### ISSUE TITLE
NVDA is not announcing the state selected for tree item

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15406]
### Steps to reproduce:

1. Open the link https://codepen.io/dpaquette/pen/WNLoEwp
2. Run NVDA
3. Tab to Table present on the screen.
4. use arrow keys to move to the select row in the table and check if the NVDA is announcing the selected state of the row.

### Actual behavior:
NVDA is not announcing the selected state of the tree item when navigating to it using arrow keys.

### Expected behavior:
NVDA should be announcing the selected state of the tree item when navigating to it using arrow keys.

### NVDA logs, crash dumps and other attachments:
[NVDA state Issue.webm](https://github.com/nvaccess/nvda/assets/93735775/1cc087e3-6d26-4cb4-a0b1-04874ec0fb6d)

#### Does the issue still occur after restarting your computer? 
Yes

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

**Comment by Brian1Gaff** [comment]
what add ons are running?
 Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.

**Comment by seanbudd** [comment]
Closing as duplicate of your issue #14352

**Comment by Adriani90** [comment]
This is also a duplicate of #11986. I think NVDA does not support aria selected yet, but I am not sure.
